sim-lib/src/lnd.rs

```diff
@@ -60,7 +62,27 @@ impl LightningNode for LndNode {
         unimplemented!()
     }
 
-    async fn track_payment(&self, _hash: PaymentHash) -> Result<(), LightningError> {
-        unimplemented!()
+    async fn track_payment(&self, hash: PaymentHash) -> Result<PaymentOutcome, LightningError> {
+        let mut client = self.client.clone();
+        let router_client = client.router();
+
+        let response = router_client
+            .track_payment_v2(TrackPaymentRequest {
+                payment_hash: hash.0.to_vec(),
+                no_inflight_updates: false,
```

Comment:
I think we want this to be true, in-flight can be a little spammy and we generally only care about success/failure here (for now).

Reply:
Did you mean false @carlaKC? Otherwise this hasn't change

Reply:
should be `false`